staging/knative/charts/serving/values.yaml

```diff
@@ -1,3 +1,13 @@
 global:
   serviceLabels: {}
 domain: example.com
+
+customMetricsApiservice:
+  enabled: true
+
+gateway:
+  https:
+    enabled: true
+
+namespace:
```

Comment:
Could we get a more descriptive name for both the resource  file and this field. To what namespace will these additional labels be created for?

Reply:
I renamed this field to `namespaceKnativeServing`. I did not rename the namespace resource file because it is already this directory /staging/knative/charts/serving/templates/namespace.yaml, which seems explicit.

> To what namespace will these additional labels be created for?
for the `knative-serving` namespace only.